packages/core/src/ceramic.ts

```diff
@@ -381,7 +381,8 @@ export class Ceramic implements CeramicApi {
 
       if (
         (networkOptions.name == Networks.MAINNET || networkOptions.name == Networks.ELP) &&
-        !(anchorServiceUrl in [DEFAULT_ANCHOR_SERVICE_URLS[networkOptions.name], "https://cas-internal.3boxlabs.com"])
+        (anchorServiceUrl !== "https://cas-internal.3boxlabs.com") &&
```

Comment:
True. Failed to see `in` misbehaviour.

Suggestion:
`!([DEFAULT_ANCHOR_SERVICE_URLS[networkOptions.name], "https://cas-internal.3boxlabs.com"].includes(anchorServiceUrl))`

Maybe also one should sanitise anchorServiceUrl on line 380 by `config.anchorServiceUrl.replace(/\/$/, '')` Do not know if it is sanitised somewhere before.